packages/ui/src/Button.module.scss

```diff
@@ -128,6 +128,17 @@
     }
   }
 
+  // Kind === danger
+  &.danger {
```

Comment:
@jankoritak could you request some hover, focus and active colors from @pawelklasa ?

Reply:
Requested

Reply:
The designs have landed in Figma. Will tweak the styles to match it.